app/model/frontsapi.scala

```diff
@@ -421,8 +421,8 @@ trait UpdateActionsTrait extends Logging {
       collectionId: String,
```

Comment:
I'm confused about the relation between the config `groups` field and the trail metadata `group` field.